pages/common/nms.md

```diff
@@ -0,0 +1,20 @@
+# nms
+
+> Command line tool that recreates the famous data decryption effect seen in the 1992 movie Sneakers from stdin.
+> More information: <https://github.com/bartobri/no-more-secrets>.
+
+- Decrypt text after a keystroke:
+
+`echo "{{Hello, World!}}" | nms`
+
+- Decrypt output, automatically:
```

Comment:
Hey there - welcome to tldr-pages! Great first page here. Reading the README for the project behind this command, the `automatically` does seem to hold meaning here. What about this instead to aid clarity?

```suggestion
- Decrypt output immediately, without waiting for a keystroke:
```

Reply:
Hi @sbrl, thanks for the welcome! I like your suggestion quite a lot, what about you @bl-ue? Would it settle the debate?

Reply:
Yeah, sounds like the user doesn't need to trigger the animation with `space`, or any key. Makes sense.

Reply:
+1 for @sbrl's suggestion—kills all the birds with one stone :D